lively.ide/studio/top-bar.cp.js

```diff
@@ -135,6 +135,20 @@ export class TopBarModel extends ViewModel {
       bindings: {
         get () {
           return [
+            { target: 'undo button', signal: 'onMouseDown', handler: () => this.target.execCommand('undo') },
+            { target: 'redo button', signal: 'onMouseDown', handler: () => this.target.execCommand('redo') },
+            { target: 'save button', signal: 'onMouseDown', handler: () => $world.execCommand('save world or project') },
```

Comment:
this unfortunately will also trigger if the only thing I do is select the drop down. So we need to introduce a custom event for *just* clicking the icon as well, its not enough to just listen for the mouse even here...